onnxruntime/core/mlas/lib/compute.cpp

```diff
@@ -850,8 +850,19 @@ Return Value:
     const float* Input = WorkBlock->Input + n * D;
     float* Output = WorkBlock->Output + n * D;
 
+    constexpr size_t CacheLineSize = 64;
+    constexpr size_t ElementsPerCacheLine = CacheLineSize / sizeof(float);
+
     while (CountN > 0) {
 
+        //
+        // Prefetch the next row of the input buffer.
+        //
+
+        for (size_t i = 0; i * ElementsPerCacheLine < D; i++) {
+            _mm_prefetch((char*)(Input + D) + i * CacheLineSize, _MM_HINT_T0);
```

Comment:
It is an SSE intruction which is probably available everywhere by now. However, should we not follow the general pattern of platform checks and availability and avoid explicit use of instructions in this function?

Reply:
Guard by MLAS_SSE2_INTRINSICS